src/Service.Tests/CosmosTests/QueryTests.cs

```diff
@@ -472,6 +479,152 @@ ... p
             Assert.AreEqual(id, response.GetProperty("id").GetString());
         }
 
+        [TestMethod]
+        public async Task QueryWithCacheEnabledShouldReturnCachedResponse()
```

Comment:
need test comments describing what is being validated:

It looks like you're querying for a record , then changing a property in that record, performing a second query to ensure the cached record is returned because you are still within the TTL of the cache entry?

Reply:
Yes second query should return the cached entry instead of the modified one.  I'll add the comments for clarity.